cardinal/plugins.py

```diff
@@ -323,6 +375,27 @@ def _get_plugin_commands(self, instance):
 
         return commands
 
+    def _get_plugin_events(self, instance):
```

Comment:
All three of these methods should probably be updated to use the `callback` terminology instead of the `event` terminology.